Footer

AleksandraWojcikk opened the pull request:
- footer component is changed
- pink background added (instead of the pink wave)
- styles, fonts changed
- proper links added to Links
- smooth scroll added to navigation list
- responsive views added.

AleksandraWojcikk committed:
vectorFooter.svg added, Footer component to Layout added

AleksandraWojcikk committed:
wave-footer backgroundImage added to the tailiwind confid

AleksandraWojcikk committed:
names of sections changed, styles of logo, fonts changed

AleksandraWojcikk committed:
links added, develoart logo with link to proper page added, some changes in styles made

AleksandraWojcikk committed:
smooth scroll to navigation links added

AleksandraWojcikk committed:
fonts changed for Navigation and Links, some changes in positioning of these components

AleksandraWojcikk committed:
margin added

AleksandraWojcikk committed:
conclicts resolved

AleksandraWojcikk committed:
fonts color changed into darkGray

AleksandraWojcikk committed:
mobile view added

AleksandraWojcikk committed:
conflicts fixed

AleksandraWojcikk committed:
mobile view added

AleksandraWojcikk committed:
tablet view added

Gumbosh reviewed (commented):
You could add display: block to a tags to ensure they take up the entire li space, which is highlighted on hover. Also for better readability, You could add some spacing between each subsection on mobile view.

AleksandraWojcikk committed:
display: block added to all of the links

AleksandraWojcikk committed:
space between subsections added on mobile and on max-mobile width also

AleksandraWojcikk committed:
some styles changes on tablet view

AleksandraWojcikk commented:
@Gumbosh Thanks for your suggestions! Please look into my code again. I hope now it looks better. I added the styles for widths less than 480px - because on really small width there were no spaces at all. Of course I'm open for your next comments :) I added display: block to all tags with hovers also.

AleksandraWojcikk committed:
conflickts resolved
